caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -37,17 +46,25 @@ class ChartContainer extends React.Component {
           // should call callback to adjust height of chart
         },
         height: () => parseInt(this.props.height, 10) - 100,
+
+        find: (classname) => ($(this.state.selector).find(classname)),
+
       },
 
       width: () => this.chartContainerRef.getBoundingClientRect().width,
 
       height: () => parseInt(this.props.height, 10) - 100,
 
-      selector: `#${this.props.sliceContainerId}`,
+      selector: this.state.selector,
 
       done: () => {
         // finished rendering callback
       },
+
+      error: () => {
```

Comment:
when does this get called? should we surface an error to the user at this point? how are we currently handling this?